src/utils/types.ts

```diff
@@ -0,0 +1,22 @@
+import ethers, { Contract, ContractFunction } from 'ethers'
+
+export type MultiCallResponse<T> = T | null
+
+export interface PredictionsRounds {
+  epoch: ethers.BigNumber
+  startBlock: ethers.BigNumber
+  lockBlock: ethers.BigNumber
+  endBlock: ethers.BigNumber
+  lockPrice: ethers.BigNumber
+  closePrice: ethers.BigNumber
+  totalAmount: ethers.BigNumber
+  bullAmount: ethers.BigNumber
+  bearAmount: ethers.BigNumber
+  rewardBaseCalAmount: ethers.BigNumber
+  rewardAmount: ethers.BigNumber
+  oracleCalled: boolean
+}
```

Comment:
I think exporting these function returns separately is a good idea so we can use the same type with `multicall`

```
const rounds = await multicall<PredictionsRounds[]>(abi, calls)
```